src/AuthenticationService.Broker/AuthenticationService.Broker.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.2" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.9" />
     <PackageReference Include="MassTransit" Version="7.0.4" />
```

Comment:
remove MassTransit, it have in kernel, and remove it in all project where kernel having

Reply:
Fixed.